posthog/tasks/usage_report.py

```diff
@@ -323,7 +323,7 @@ def get_teams_with_event_count_in_period(begin: datetime, end: datetime) -> List
         """
         SELECT team_id, count(1) as count
         FROM events
-        WHERE timestamp between %(begin)s AND %(end)s
+        WHERE timestamp between %(begin)s AND %(end)s AND event != '$feature_flag_called'
```

Comment:
I can't comment directly on the function name because it's out of range for the PR, but this is in a function called `get_teams_with_event_count_in_period` and I wonder if we should rename the function to `get_teams_with_filtered_event_count_in_period` so that it's more obvious when looking at the function call that it won't return ALL events.